resource-timing/resource_TAO_multi.htm

```diff
@@ -48,7 +48,7 @@
 </head>
 <body>
 <h1>Description</h1>
-<p>This test validates that for a cross origin resource, the timing allow check algorithm will fail when the HTTP response includes more than one Timing-Allow-Origin header value (not overwrite any existing TAO values when appending the new value).</p>
+<p>This test validates that for a cross origin resource, the timing allow check algorithm will pass when the HTTP response has multiple Timing-Allow-Origin header fields and the subsequent field value is separated by a comma.</p>
```

Comment:
The expected behavior, at least per HTTP spec should be: timing allow check algorithm will pass when the HTTP response has multiple Timing-Allow-Origin header fields, and one of those values passes the origin check. 

Concretely, these two cases should be identical:

```
Timing-Allow-Origin: foo.com
Timing-Allow-Origin: bar.com
-- same as --
Timing-Allow-Origin: foo.com, bar.com
```

If current implementations fail this, then thats an implementation bug.

/cc @toddreifsteck